src/grt/grt-wave_opt_file-parse.adb

```diff
@@ -59,28 +60,28 @@ package body Grt.Wave_Opt_File.Parse is
                         return Boolean;
 
    -- Parse the line where the version is set
-   procedure Parse_Version (Line : String_Access);
+   procedure Parse_Version (Line : String; Line_Pos : Positive);
```

Comment:
Line_Pos sounds to be a position within Line.  For line number, Lineno is more common.  (Not a blocking comment),